roles/caddy/tasks/main.yml

```diff
@@ -46,7 +45,7 @@
 
 - name: install caddyfile
   template:
-    src: Caddyfile
+    src: "{{ caddy_file }}"
```

Comment:
without this variable, it was pulling it just the template/Caddyfile, not the local variable we were using

Reply:
Indeed, this is meant to be configurable.